docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,75 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Finding a printer that works with Tails can be challenging because Tails is
+based on the Linux operating system, which often has second-class hardware
+support in comparison to operating systems such as Windows or macOS.
+
+We `maintain a list of printers
+<https://docs.securedrop.org/en/stable/hardware.html#printers>`_ that we have
```

Comment:
Additional note: this actually matters even if you only care about building the HTML output target (e.g. for hosting on ReadTheDocs). Imagine the following scenario:

1. You are a developer working on the documentation.
2. You are editing `.rst` files in your editor and observing the corresponding changes to the HTML output in your browser (thanks to the sweet auto-reload provided by `make docs`).
3. You click this link in your local rendered copy of the HTML output.

You would want this link to go to the relevant section of *your local copy* of the HTML documentation, but instead it would take you to the externally hosted version of the documentation, which might differ significantly from your local copy (and would be confusing/incorrect behavior in any case).